.github/workflows/plugin_test.yaml

```diff
@@ -0,0 +1,102 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: test plugin
+
+on:
+  push:
+    branches: [generic_test_in_ci_cd]
```

Comment:
This should probably changed to master

Reply:
Thanks! To bring this. I that should be `*` which means all in push. This is for just now. If you approve it I will change it immediately.

Reply:
NOted!